ci/si_pipeline.sh

```diff
@@ -75,10 +74,16 @@ case $CLUSTER_LAUNCH_CODE in
       if [ ${SI_CODE} -gt 0 ]; then
         download-diagnostics-bundle
       fi
-      dcos-launch delete
+      dcos-launch delete || true
       exit "$SI_CODE" # Propagate return code.
       ;;
   2) exit-as-unstable "Cluster launch failed.";;
-  3) exit-as-unstable "Cluster did not start in time.";;
-  *) exit-as-unstable "Unknown error in cluster launch: $CLUSTER_LAUNCH_CODE";;
+  3)
+      dcos-launch delete || true
```

Comment:
I dunno whether we want to keep this or not - I guess I don't have strong opinion, although I would probably remove these...

Reply:
I've removed the `|| true`. We do have to keep the delete, though, since a rollback does not cleanup everything.